compiler/test/dotty/tools/dotc/vulpix/ChildMain.scala

```diff
@@ -0,0 +1,83 @@
+package dotty.tools.dotc
+package vulpix
+
+import java.io.{
+  File => JFile,
+  InputStream, ObjectInputStream,
+  OutputStream, ObjectOutputStream,
+  ByteArrayOutputStream, PrintStream
+}
+import java.lang.reflect.InvocationTargetException
+
+import dotty.tools.dotc.vulpix.Statuses._
+
+object ChildMain {
+  val realStdin = System.in
+  val realStderr = System.err
+  val realStdout = System.out
+
+  private def runMain(dir: JFile): Status = {
+    def renderStackTrace(ex: Throwable): String =
+      ex.getStackTrace
+        .takeWhile(_.getMethodName != "invoke0")
```

Comment:
What is this for? I think I'd be happier not having my stack traces mangled in some undocumented way, what if I have a test with an invoke0 method?